terminology.md

```diff
@@ -207,6 +207,21 @@ console.log(abd);  // runtime error
 #### References
 - [early error](https://tc39.es/ecma262/#early-error) defined in the ECMAScript specification editor's draft.
 
+### Options Bag
+
+#### Definition
+
+A parameter value that is a JavaScript object with properties looked up eagerly. It is used to configure behavior of a function. Methods on the options bag object should be invoked with `undefined` as the default value for `this`
```

Comment:
How do you mean eagerly? Not quite sure what can be lazy here.

Reply:
@rkirsling unclear a bit on exact semantic! just following plenary description.

CC: @erights @ljharb

Reply:
I think we can drop " looked up eagerly"

Reply:
I would add here: "...a JavaScript object with a set of optional specification defined properties."

Reply:
We shouldn’t drop it. There is a big difference between eager lookup and what Proxy does, lazy lookup.

Reply:
live (on every use), lazy (on first use), and eager (prior to any use) might be good to define.